src/Sulu/Bundle/DocumentManagerBundle/Bridge/DocumentInspector.php

```diff
@@ -110,9 +111,17 @@ public function getContentPath(PathBehavior $document)
      */
     public function getStructureMetadata(StructureBehavior $document)
     {
+        $alias = $this->getMetadata($document)->getAlias();
+        $structureType = $document->getStructureType();
+        if (null === $structureType) {
+            $webspace = $this->webspaceManager->findWebspaceByKey($this->getWebspace($document));
+            $structureType = $webspace->getTheme()->getDefaultTemplate($alias);
+            $document->setStructureType($structureType);
```

Comment:
For me that's also unexpected behavior. Just asking for the metadata should not automatically set it.